seed/seed20.go

```diff
@@ -62,6 +63,7 @@ type seed20 struct {
 	auxInfos map[string]*internal.AuxInfo20
 
 	snaps             []*Snap
+	modes             [][]string
```

Comment:
Is this expect to match the length of the array of snaps (that is, for each snap there's an array of modes that go with it?). If so can you please add a comment in the code.